File: backend-csharp/Models/PhuongThucThanhToan.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class PhuongThucThanhToan
{
    public long Id { get; set; }

    public string TenPhuongThuc { get; set; } = null!;

    public string Ma { get; set; } = null!;

    public string Loai { get; set; } = null!;

    public string? LogoUrl { get; set; }

    public string? CauHinh { get; set; }

    public decimal PhiThanhToan { get; set; }

    public string TrangThai { get; set; } = null!;

    public virtual ICollection<GiaoDichThanhToan> GiaoDichThanhToans { get; set; } = new List<GiaoDichThanhToan>();
}

File: backend/models/PhuongThucThanhToan.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/db");

const PhuongThucThanhToan = sequelize.define(
  "PhuongThucThanhToan",
  {
    id: { type: DataTypes.BIGINT, primaryKey: true, autoIncrement: true },
    ten_phuong_thuc: { type: DataTypes.STRING(100), allowNull: false },
    ma: { type: DataTypes.STRING(50), allowNull: false },
    loai: { type: DataTypes.STRING(20) },
    logo_url: { type: DataTypes.STRING(255) },
    cau_hinh: { type: DataTypes.TEXT },
    phi_thanh_toan: { type: DataTypes.DECIMAL(5, 2), defaultValue: 0 },
    trang_thai: { type: DataTypes.STRING(20), defaultValue: "active" },
  },
  {
    tableName: "PhuongThucThanhToan",
    timestamps: false,
  },
);

module.exports = PhuongThucThanhToan;
